great_expectations/experimental/datasources/postgres_datasource.py

```diff
@@ -9,7 +9,8 @@ class PostgresDatasource(SQLDatasource):
 
     Args:
         name: The name of this postgres datasource
-        connection_str: The SQLAlchemy connection string used to connect to the postgres database.
+        connection_string: The SQLAlchemy connection string used to connect to the postgres
```

Comment:
To line this up with the actual variable names `conntection_str` got changed to `connection_string`.